torch/ao/quantization/pt2e/prepare.py

```diff
@@ -365,11 +365,13 @@ def _maybe_insert_input_observers_for_node(
         )
         new_args.append(new_arg)
 
-    # Clone has a memory_format kwarg and zeros_like has a pin_memory kwarg
-    # that persist in exported graph. This is just a work around for these.
+    # Clone has a memory_format kwarg, zeros_like has a pin_memory kwarg, and
+    # gelu has a memory kwarg that persist in exported graph.
```

Comment:
nit: the kwarg of gelu is not `memory` right? should be something relates with `approximate`?

Reply:
@leslie-fang-intel  Thank you for pointing out! Can I edit the comment as "# Clone has a memory_format kwarg, zeros_like has a pin_memory kwarg and gelu has kwarg of approximate  that persist in exported graph. This is just a work around for these." ?